docs/examples/move/nft_marketplace/README.md

```diff
@@ -0,0 +1,179 @@
+# Marketplace Guide
+
+The `marketplace_extension.move` module provides a straightforward implementation of a marketplace extension. To utilize it, follow the steps outlined below.
+The `item_for_market.move` contains mocked item data for use within the marketplace.
+The `rental_extension.move` is an extension adds functionality to enable item rentals.
+
+## Steps to Use the Marketplace
+
+### 1. Connect to the Network
+
+Connect to the IOTA network (e.g., using a faucet to obtain tokens).
+
+### 2. Install Kiosk
+
+By installation, we mean creating a Kiosk object and an OwnerCap, then transferring them to the caller.
+Run the following command to install the Kiosk module:
+
+```bash
+iota client call \
+    --package 0x2 \
+    --module kiosk \
+    --function default
+```
+
+### 3. Publish `nft_marketplace` package
+
+#### 3.1(Optional) Publish Kiosk rules modules if these are not present in the network you are using
+
+Publish Kiosk rules modules(package):
+
+```bash
+iota client publish iota/kiosk
```

Comment:
Should this command be changed like the following?
```
iota client publish ./kiosk
```
or
```
iota client publish $IOTA_REPO_DIR"/kiosk"
```
I mean does it refer to the `kiosk` folder in the root of the repository?
I think this would be nice to clarify.

Reply:
I prefer `$IOTA_REPO_DIR` option